components/CarComponentOption.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CarOption from "./CarOption";
+
+const CarComponentOption = ({ name, state, clickHandler, tooltipText }) => {
+  let classNames = `uppercase mx-1 my-1 py-2 px-4 border border-white
+  rounded-md part-button text-sm font-medium transition-colors duration-200`;
+
+  if (state === "active") {
```

Comment:
Try styled components :)

Reply:
Already tried ;) In most of my projects I use styled components. This project was my experiment on tailwind with react